src/triage/util/pandas.py

```diff
@@ -16,10 +17,13 @@ def downcast_matrix(df):
     and save memory on the index storage.
     """
     logging.debug("Downcasting matrix. Starting memory usage: %s", df.memory_usage())
-    new_df = (
-        df.apply(partial(pd.to_numeric, downcast="float"))
-        .apply(partial(pd.to_numeric, downcast="integer"))
-    )
+
+    logging.debug(df.dtypes)
+
+    new_df = df.apply(lambda x: x.astype(np.float32))
```

Comment:
I'm honestly unsure -- is this functionally different from `df.astype(np.float32)`? [[docs]](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.astype.html)

(I ask only because I imagine that might be implemented in C and more performant.)

Reply:
Also, perhaps I missed it, but is there a test update to go along with this, (to ensure that we're addressing what the last version didn't address)?

Reply:
(bump)